src/screens/Admin/screens/InstructorPulse/index.js

```diff
@@ -10,13 +12,15 @@ export default ({instructors}) => (
     main={
       <InstructorsList
         title={unpublishedTitleText}
-        instructors={filter(instructors, ['published_lessons', 0])}
+        instructors={filter(instructors, ['published_lessons', 0])
+          .sort(sortBy('-submitted_lessons', '-claimed_lessons', '-approved_lessons', '-id'))}
```

Comment:
Ah, ok I see why you used this now instead of lodash. Cool, looks good! And this will be fine to move this view into the root and allow all to see it because this data is only returned from the response if the instructor has access to it, so there is no risk of other instructors seeing royalty numbers and stuff like that, right?